packages/components/package.json

```diff
@@ -65,7 +65,7 @@
     "@babel/eslint-parser": "^7.22.15",
     "@babel/plugin-proposal-decorators": "^7.23.0",
     "@ember/optional-features": "^2.0.0",
-    "@ember/test-helpers": "^3.2.0",
+    "@ember/test-helpers": "^2.9.3",
```

Comment:
note: we should be updating `ember-a11y-testing` to allow `@ember-test-helpers` `3.x`

Reply:
resolved in [f81971a](https://github.com/hashicorp/design-system/pull/1704/commits/f81971a910bcf42439dc669e1be3a24228fb39f3)